src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java

```diff
@@ -136,9 +135,13 @@
     }
     
     public static class SecurityGroupPlugin {
```

Comment:
Rename this inner class just to SecurityGroup.

Reply:
Done